deploy.go

```diff
@@ -197,7 +191,7 @@ func (gc *GatewayClient) GetVersion() (string, error) {
 	case resp == nil:
 		return "", errNilReponse
 	case !(resp.StatusCode >= 200 && resp.StatusCode <= 299):
-		return "", nil
+		return "", errors.New("Error: " + resp.Status)
```

Comment:
Error messages should start with a small letter. The message could be more descriptive, e.g. `fmt.Errorf("error response: %s", resp.Status)`.

Reply:
Fixed